client/src/components/HeaderCell.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Table, Icon } from "semantic-ui-react";
+
+function HeaderCell({ handleHeaderCellClick, headerCell, id, sortIconName }) {
+  const { header, sortKey } = headerCell;
+  return (
+    <Table.HeaderCell
+      onClick={() => handleHeaderCellClick(id, sortKey)}
+      id={id}
+    >
```

Comment:
You can also change the cursor so the user knows they can click on the column name, if you use in css something like this: `cursor: pointer`.

Reply:
Done